common/config/webpack/sampleapp.webpack.config.js

```diff
@@ -43,7 +43,8 @@ const webpackConfig = (sampleAppDir, env, babelConfig) => {
           options: {
             transpileOnly: true
           },
-          exclude: /dist/
+          exclude: /dist/,
+          sideEffects: false
```

Comment:
Let's add these details to the storybook/documentation user story so we won't forget to document this for customers